site/content/en/latest/tasks/observability/proxy-accesslog.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Proxy Access Log"
```

Comment:
different terms are used in this doc, like `access log` & `access logs`, should be consistent, prefer `access logs`